core/telemetry/src/worker/node.rs

```diff
@@ -76,6 +78,7 @@ impl<TTrans: Transport> Node<TTrans> {
 			addr,
 			socket: NodeSocket::ReconnectNow,
 			transport,
+			connection_timeout: None,
```

Comment:
Why is this in `NodeSocket` and not in `NodeSocketConnected`? It is only ever user or relevant when we're connected.